OCABundles/Ep31SvFAetugFPe5CGzJxt/Ep31SvFAetugFPe5CGzJxt:2:Attestation_numérique_didentité_gouvernemental_(EXP):1.0/testdata.csv

```diff
@@ -0,0 +1,2 @@
+Nom, Prénom, expiry_date
```

Comment:
Should have data for all of the fields in the schema and capture base:

"Photo”, "Nom”, "Date d'expiration”, "Prénom”, "Nom du parent 1”,  "Date de naissance”,  "Date d'émission”,  "Niveau d'identification”, "Nom du parent 2"